CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## Version 4.0.4
+
+* Add support for dropping keyspace(s) - Issue #469
```

Comment:
This was not supported in previous versions.

Reply:
This feels less like a missing feature and more like a bug to me. However, even if it is a new feature,  then we should state that in the documentation of older versions as a limitation. The consequences of hitting this are pretty annoying for the end user.

Reply:
I have added a note about this in #464

Reply:
After some testing I've noticed that we had support for it before we stepped datastax driver version. So this needs to be fixed on master only since driver step was released in 4.0.

Reply:
That's good news.  But then I suppose we should change the commit message and the changelog to reflect that we are fixing a bug here - Aka, that behavior has changed and was corrected.